app/views/travel_advice/index.html.erb

```diff
@@ -33,7 +33,7 @@
                 Check the <a href="/guidance/national-lockdown-stay-at-home">rules that apply to you in England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/">Scotland</a>, <a href="https://gov.wales/covid-19-alert-levels">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you">Northern Ireland</a>.
               </p>
               <p class="govuk-body">
-                From 4am on 18 January, if you intend to travel to <a href="/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england">England</a>, or from 4am on 15 January if you intend to travel to <a href="https://www.gov.scot/news/pre-departure-coronavirus-testing-update/">Scotland</a>, including UK nationals returning home from travel abroad, you must provide evidence of a negative COVID-19 test result taken up to 3 days before departure. If you do not comply (and you do not have a valid exemption) your airline or carrier may refuse you boarding and/or you may be fined on arrival. All other current entry requirements and restrictions continue to apply.
+                From 4am on 18 January, if you intend to <a href="/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england">travel to England</a>, <a href="https://www.gov.scot/publications/coronavirus-covid-19-public-health-checks-at-borders/pages/testing-for-people-travelling-to-scotland/">Scotland</a>, or <a href="https://gov.wales/new-mandatory-coronavirus-testing-for-travellers-into-wales">Wales</a>, including UK nationals returning home from travel abroad, you must provide evidence of a negative COVID-19 test result taken up to 3 days before departure. If you do not comply (and you do not have a valid exemption) your airline or carrier may refuse you boarding and/or you may be fined on arrival. All other current entry requirements and restrictions continue to apply.
```

Comment:
```
to travel to <a href="/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england">England</a>
```

Reply:
They specifically asked for this to be changed for some accessibility thing. I'll add it to the commit message